libbeat/_meta/fields.common.yml

```diff
@@ -11,6 +11,11 @@
         set in the configuration file, then that value is used. If it is not
         set, the hostname is used. To set the Beat name, use the `name`
         option in the configuration file.
+      format: url
+      params: 
+        urlTemplate: "../app/kibana#/dashboard/79ffd6e0-faa0-11e6-947f-177f697178b8?_a=(query:(query_string:(analyze_wildcard:!t,query:'beat.hostname:{{value}}')))"
```

Comment:
Would it be possible to only apply this to the `metricbeat-*` index pattern instead of all beats?